### PR TITLE
Fix Reference docs and sync it with Examples dir

### PR DESCRIPTION
**Pull request by aLekSer**
There was a discrepancy between website [agones.dev](http://agones.dev/) and Examples dir
yaml files content.
For #969.

### CODE REVIEW

**Comment by aLekSer** [comment]
I have made a copy of yaml files as is, because without comments it would be harder to keep them in sync later.

**Comment by aLekSer** [comment]
```
Step #17: --- FAIL: TestGameServerUnhealthyAfterDeletingPod (88.45s)
Step #17: gameserver_test.go:211: 
Step #17: Error Trace:	gameserver_test.go:211
Step #17: Error: Received unexpected error:
Step #17: timed out waiting for the condition
Step #17: waiting for GameServer to be Unhealthy default/udp-serverjrtf8
```

File: site/content/en/docs/Reference/fleet.md [hunk 1]
```diff
@@ -129,7 +150,6 @@ A `FleetAllocation` is used to allocate a `GameServer` out of an existing `Fleet
 A full `FleetAllocation` specification is available below and in the 
 {{< ghlink href="examples/fleetallocation.yaml" >}}example folder{{< /ghlink >}} for reference:
 
-{{% feature expiryVersion="0.12.0" %}}
```
Comment (roberthbailey) [inline site/content/en/docs/Reference/fleet.md]: Why did you remove this part?

File: site/content/en/docs/Reference/gameserver.md [hunk 2]
```diff
@@ -42,24 +42,51 @@ spec:
...
+  # https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/#podtemplate-v1-core
   template:
+    # pod metadata. Name & Namespace is overwritten
     metadata:
       labels:
         myspeciallabel: myspecialvalue
+    # Pod Specification
     spec:
       containers:
       - name: example-server
```
Comment (roberthbailey) [inline site/content/en/docs/Reference/gameserver.md]: should we change this to the simple udp server?

File: site/content/en/docs/Reference/fleet.md (hunk 1 above)
Comment (aLekSer) [reply site/content/en/docs/Reference/fleet.md]: You can see it was second and redundant part, it has one closing tag and two expiryVersion above.

Comment (aLekSer) [reply site/content/en/docs/Reference/fleet.md]: line 121 {{% feature expiryVersion="0.12.0" %}}
## Fleet Allocation Specification

File: site/content/en/docs/Reference/gameserver.md (hunk 2 above)
Comment (aLekSer) [reply site/content/en/docs/Reference/gameserver.md]: Yes, that will be more clear, will update

File: site/content/en/docs/Reference/fleet.md (hunk 1 above)
Comment (aLekSer) [reply site/content/en/docs/Reference/fleet.md]: https://github.com/googleforgames/agones/blame/0b6749d69fbcd23384aaf42f6a61aa3740c5e380/site/content/en/docs/Reference/fleet.md#L121
I cannot find the exact commit were it happens. But as you could see from link above.
There are two opening `expiryVersion`  with one closing.
Also at https://github.com/googleforgames/agones/blame/0b6749d69fbcd23384aaf42f6a61aa3740c5e380/site/content/en/docs/Reference/fleet.md#L132

Comment (aLekSer) [reply site/content/en/docs/Reference/fleet.md]: So there were a nested feature blocks. And as long we don't need FleetAllocation any more we don't need to publish anything about it.

Comment (roberthbailey) [reply site/content/en/docs/Reference/fleet.md]: makes sense, thanks. that was probably my doing, so thank you for fixing it.
